src/Component/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import cn from 'classnames';
+import { FilterBy } from '../../types/FilterBy';
+
+type Props = {
+  filterValue: FilterBy;
+  activeTodosCount: number;
+  complitedTodosCount: number;
+  setFilterValue: (value: FilterBy) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodosCount, complitedTodosCount, filterValue, setFilterValue,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filterValue === FilterBy.ALL,
+          })}
+          onClick={() => setFilterValue(FilterBy.ALL)}
+        >
+          {FilterBy.ALL}
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', {
+            selected: filterValue === FilterBy.ACTIVE,
+          })}
+          onClick={() => setFilterValue(FilterBy.ACTIVE)}
+        >
+          {FilterBy.ACTIVE}
+        </a>
+
+        <a
+          href="#/completed"
```

Comment:
It's recommended to prevent the default action for the anchor tag. You should add event.preventDefault() in the onClick handler to prevent page refresh/reload when the link is clicked.